adapters/repos/db/lsmkv/bucket.go

```diff
@@ -163,6 +177,17 @@ func (b *Bucket) Get(key []byte) ([]byte, error) {
 	return b.disk.get(key)
 }
 
+// GetBySecondary retrieves an object using one of its secondary keys. A bucket
+// can have an infinite number of secondary keys. Specify the secondary key
+// position as the first argument.
```

Comment:
what is the position of secondary key?

Reply:
It's basically the index. Let's say you have 2 secondary keys, then you'd basically have independent secondary indexes at position 0 and at position 1.